c/tskit/tables.c

```diff
@@ -33,6 +34,7 @@
 #include <math.h>
 
 #include <tskit/tables.h>
+#include <tskit/trees.h>
```

Comment:
We don't want to include trees.h here - the idea is that you don't have to compile the trees code if you're only interested in the tables API.

Reply:
I "fixed" this by moving the relevant defines to tables.h.  I think I got the doxygroup stuff right?